File: nightgrass/src/main/java/com/goodfeel/nightgrass/repo/OrderItemRepository.java
package com.goodfeel.nightgrass.repo;

import com.goodfeel.nightgrass.data.OrderItem;
import org.springframework.data.repository.reactive.ReactiveCrudRepository;
import reactor.core.publisher.Flux;

public interface OrderItemRepository extends ReactiveCrudRepository<OrderItem, Long> {
    Flux<OrderItem> findByOrderId(Long orderId);
}

File: nightgrass/src/main/kotlin/com/goodfeel/nightgrass/repo/OrderItemRepository.kt
package com.goodfeel.nightgrass.repo

import com.goodfeel.nightgrass.data.OrderItem
import org.springframework.data.repository.reactive.ReactiveCrudRepository
import reactor.core.publisher.Flux

interface OrderItemRepository : ReactiveCrudRepository<OrderItem, Long> {
    fun findByOrderId(orderId: Long): Flux<OrderItem>
}
